charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+      force: true  # Force forces resource updates through a replacement strategy.
+      cleanupOnFail: true  # Allows deletion of new resources created during the Helm upgrade action when it fails.
+      remediation:
+        retries: 3  # Retries is the number of retries that should be attempted on failures before givin up. An uninstall is performed between each attempt.
+        strategy: rollback  # rollback or uninstall.
+
+# -- Values for the Flux chat. Ref.: https://github.com/fluxcd-community/helm-charts/blob/flux2-2.10.2/charts/flux2/values.yaml
+# @default -- See `values.yaml`
+flux2:
+  # -- Enable or disable FluxCD completely. New Relic' Super Agent need Flux to work, but the user can use an already
+  # existing Flux deployment. With that use case, the use can disable Flux and use this chart to only install the CRs
+  # to deploy the Super Agent.
+  enabled: true
+
+  # -- The installation of the CRDs is managed by the chart itself.
+  installCRDs: true
+  # -- This is the domain name of the cluster.
+  clusterDomain: cluster.local
+
+  # -- Source controller provides a way to fetch artifacts to the rest of controllers. The source API (which reference
+  # [can be read here](https://fluxcd.io/flux/components/source/api/v1/)) is used by admins and various automated
+  # operators to offload the Git, OCO, and Helm repositories management.
+  # @default -- Enabled
+  sourceController:
+    create: true
+    annotations:
+      prometheus.io/port: "8080"
+      prometheus.io/scrape: "false"  # TODO: Should we create a FluxCD entity to monitor flux itself!?
+
+  # -- Helm controller is a Kubernetes operator that allows to declaratively manage Helm chart releases with Kubernetes
+  # manifests. The Helm release is defined in a CR ([Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources))
+  # named `HelmRelease` that the operator will reconcile on the apply, edit, or deletion of a `HelmRelease` resource.
+  #
+  # New Relic' Super Agent will use this controller by creating `HelmRelease` CRs based in the configuration stored on
+  # OpAmp.
+  # @default -- Enabled
+  helmController:
+    create: true
+    annotations:
+      prometheus.io/port: "8080"
+      prometheus.io/scrape: "false"  # TODO: Should we create a FluxCD entity to monitor flux itself!?
+
+  # -- This controller is exactly the same as the Helm controller (refer to it) but assembling manifests with
+  # Kustomize instead of using templating systems like Helm.
+  # 
+  # New Relic' Super Agent will use this controller by creating `Kustomization` CRs based in the configuration stored
+  # on OpAmp.
+  # @default -- Enabled
+  kustomizeController:
+    create: true
+    annotations:
+      prometheus.io/port: "8080"
+      prometheus.io/scrape: "false"  # TODO: Should we create a FluxCD entity to monitor flux itself!?
```

Comment:
it is not used yet I would not deploy it for now

Reply:
Fair. Removed on [#244a4b4](https://github.com/newrelic/helm-charts/pull/1175/commits/244a4b4c444f4b09068dd3d939dcc099571153f6)